logstash-core/src/test/java/org/logstash/instrument/witness/process/ProcessWitnessTest.java

```diff
@@ -55,7 +56,7 @@ public void testRefreshChanges() throws InterruptedException {
         ProcessWitness.Snitch snitch = witness.snitch();
         assumeTrue(ProcessWitness.isUnix);
         witness.refresh();
-        long before = snitch.cpuProcessPercent();
+        short before = snitch.cpuProcessPercent();
```

Comment:
@jakelandis :D why `short` and make code inspections cry? LGTM otherwise :)

Reply:
The method returns a short, and trying to avoid implicit casts for the comparison below.  I should change the method sig to return not-short. 

thanks, will wait for CI before merge.